tests/components/gardena_bluetooth/test_switch.py

```diff
@@ -0,0 +1,84 @@
+"""Test Gardena Bluetooth sensor."""
+
+
+from unittest.mock import Mock, call
+
+from gardena_bluetooth.const import Valve
+import pytest
+from syrupy.assertion import SnapshotAssertion
+
+from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_TURN_OFF,
+    SERVICE_TURN_ON,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+
+from . import setup_entry
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_switch_chars(mock_read_char_raw):
+    """Mock data on device."""
+    mock_read_char_raw[Valve.state.uuid] = b"\x00"
```

Comment:
Fine as-is, but I'd probably make a copy of `Valve.remaining_open_time` as `remaining_open_time` as it might make this a bit more readable.

Reply:
Noted. Will see if i refactor.